ios/FluentUI/AvatarGroup/AvatarGroup.swift

```diff
@@ -218,13 +222,23 @@ public struct AvatarGroup: View {
 
         func path(in rect: CGRect) -> Path {
                 var cutoutFrame = Rectangle().path(in: rect)
-                cutoutFrame.addPath(Circle().path(in: CGRect(x: xOrigin,
-                                                             y: yOrigin,
-                                                             width: cutoutSize,
-                                                             height: cutoutSize)))
+            cutoutFrame.addPath(Circle().path(in: CGRect(x: xOrigin,
+                                                         y: yOrigin,
+                                                         width: cutoutSize,
+                                                         height: cutoutSize)))
                 return cutoutFrame
         }
     }
+
+    // Detects if language code direction is left to right
+    private func isLeftToRight() -> Bool {
```

Comment:
should this function live here? I feel like it is useful to other controls in the future.

Reply:
Good point, I'm thinking maybe it can live in one of the Core files instead. Would you recommend a specific existing file or should I create a new helpers method file?

Reply:
Will make a Locale extension per @mischreiber 's suggestion